extension/EdFi.Ods.Extensions.LMSX/LMS-Harmonizer/1090-lms-assignment-submissions.sql

```diff
@@ -41,9 +42,52 @@ BEGIN
 		ON EDFISTUDENT.Id = lmsUser.EdFiStudentId
     WHERE lmsSubmission.SourceSystem = @SourceSystem
 
+	-- Here we are building missing submissions when they are not present
+	-- We do this to standardize what is shown in the visualizations
+	ALTER TABLE
+		#ALL_SUBMISSIONS
+	ALTER COLUMN
+		AssignmentSubmissionIdentifier
+	int NULL;
+
+
+	INSERT INTO #ALL_SUBMISSIONS
+	SELECT
+		edfisectionassociation.SectionIdentifier
+			+ '#' + lmsxassignment.AssignmentIdentifier
```

Comment:
And shouldn't this be the `SourceSystemIdentifier` for the assignment?

Reply:
Oh, I missed the `x`! I was thinking `lmsAssignment.SourceSystemIdentifier`, and you put in the `lmsXassignment.AssignmentIdentifier`, which are the same thing. So this is fine. Just all too easy to overlook the X.